tutorials/2d/custom_drawing_in_2d.rst

```diff
@@ -69,12 +69,14 @@ redrawn if modified:
 
     extends Node2D
 
-    export (Texture) var texture setget _set_texture
+    @export var texture : Texture :
```

Comment:
```suggestion
    @export var texture: Texture:
```

See [GDScript style guide](https://docs.godotengine.org/en/latest/tutorials/scripting/gdscript/gdscript_styleguide.html#declared-types) and [the example](https://docs.godotengine.org/en/latest/tutorials/scripting/gdscript/gdscript_basics.html#properties-setters-and-getters).